photon-client/src/main/scala/com/linkedin/photon/ml/cli/game/training/Driver.scala

```diff
@@ -14,105 +14,105 @@
  */
 package com.linkedin.photon.ml.cli.game.training
 
-import scala.collection.Map
+import scala.util.{Failure, Success, Try}
 
 import org.apache.hadoop.fs.Path
 import org.apache.spark.SparkContext
+import org.apache.spark.mllib.linalg.Vector
 import org.apache.spark.sql.DataFrame
 import org.slf4j.Logger
 
 import com.linkedin.photon.ml.SparkContextConfiguration
+import com.linkedin.photon.ml.Types.{FeatureShardId, IndexMapLoaders}
 import com.linkedin.photon.ml.avro.model.ModelProcessingUtils
 import com.linkedin.photon.ml.cli.game.GAMEDriver
 import com.linkedin.photon.ml.data._
 import com.linkedin.photon.ml.estimators.{GameEstimator, GameParams}
 import com.linkedin.photon.ml.evaluation.Evaluator.EvaluationResults
-import com.linkedin.photon.ml.io.ModelOutputMode
+import com.linkedin.photon.ml.io.{GLMSuite, ModelOutputMode}
 import com.linkedin.photon.ml.model.GAMEModel
+import com.linkedin.photon.ml.normalization.{NormalizationContext, NormalizationType}
+import com.linkedin.photon.ml.stat.BasicStatisticalSummary
+import com.linkedin.photon.ml.util.Implicits._
 import com.linkedin.photon.ml.util._
+import com.linkedin.photon.ml.util.Utils._
 
 /**
- * The driver class, which provides the main entry point to GAME model training.
+ * The Driver class, which drives the training of Game model.
+ * Note: there is a separate Driver to drive the scoring of Game models.
```

Comment:
In "header comments" I fixed to `@note` in the whole code base, but in one liner comments I kept `// NOTE` uniformly.